libtenzir/builtins/operators/python.cpp

```diff
@@ -468,22 +468,42 @@ class plugin final : public virtual operator_plugin<python_operator>,
 
   auto initialize(const record& plugin_config, const record& global_config)
     -> caf::error override {
+    auto is_node = try_get_or<bool>(global_config, "tenzir.is-node", false);
+    if (!is_node) {
+      return is_node.error();
+    }
     auto create_virtualenv
       = try_get_or<bool>(plugin_config, "create-venvs", true);
     if (!create_virtualenv) {
       return create_virtualenv.error();
     }
+    auto venvs_subdir = is_node ? "venvs" : "client-venvs";
```

Comment:
What do you think about generally forcing the cache directory to be `$TMPDIR` when we're running `tenzir` instead of `tenzir-node`? Then we also don't need to change the Python operator's location, and we can freely wipe the entire cache directory every time the node starts up.

Reply:
This sounds reasonable to me.